contrib/helloimage/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM docker://docker.io/library/alpine:latest
+
+WORKDIR /tmp
+
```

Comment:
Should we run this as user, e.g. add `USER 1000`? We do not want to run things as root.

Reply:
Done

Reply:
Actually, I'm pulling that out.  Once I added it, I got this error when running the image from quay.io.

```
Copying blob acab339ca1e8 done  
ERRO[0002] Error while applying layer: ApplyLayer exit status 1 stdout:  stderr: potentially insufficient UIDs or GIDs available in user namespace (requested 0:12 for /var/spool/mail): Check /etc/subuid and /etc/subgid: lchown /var/spool/mail: invalid argument 
Error: writing blob: adding layer with blob "sha256:ee0cde9de8a68f171a8c03b0e9954abf18576947e2f3187e84d8c31ccd8f6a09": ApplyLayer exit status 1 stdout:  stderr: potentially insufficient UIDs or GIDs available in user namespace (requested 0:12 for /var/spool/mail): Check /etc/subuid and /etc/subgid: lchown /var/spool/mail: invalid argument
```

I think we should be OK without setting the user.